src/libexpr/eval.hh

```diff
@@ -6,6 +6,7 @@
 #include "symbol-table.hh"
 #include "hash.hh"
 #include "config.hh"
+#include "flake/flakeref.hh"
```

Comment:
Same.

Reply:
This import is necessary here, the `FlakeRef` concept is used now because `registryOverrides` are stored in the `EvalState`.